packages/autocomplete-core/src/getPropGetters.ts

```diff
@@ -112,6 +112,12 @@ export function getPropGetters<TItem, TEvent, TMouseEvent, TKeyboardEvent>({
           setStatus,
           setContext,
           event,
+        };
+        props.onSubmit(onSubmitPayload);
+        props.plugins.forEach((plugin) => {
```

Comment:
It'd be better to implement the plugins forwarding in `getDefaultProps`. It'll make sure that we don't forget occurrences and it's easier to keep track. Wdyt?

Reply:
Yes, it makes sense. https://github.com/algolia/autocomplete.js/pull/316/commits/88fa4c1db1fcb4acdfd55f39eeb6c96949bd6190